server/sse_test.go

```
@@ -208,8 +192,17 @@ func TestSSEServer(t *testing.T) {
 				}
 				defer resp.Body.Close()
 
+				endpointEvent, err = readSeeEvent(sseResp)
+				if err != nil {
+					t.Fatalf("Failed to read SSE response: %v", err)
```

golangci-lint (1.64.8)

testinggoroutine: call to (*testing.T).Fatalf from a non-test goroutine

(govet)